imagersite/imager_images/models.py

```diff
@@ -0,0 +1,71 @@
+from django.db import models
+from django.utils.encoding import python_2_unicode_compatible
+from imager_profile.models import ImagerProfile
+from django.db.models.signals import post_delete
+from django.dispatch import receiver
+from django.contrib.auth.models import User
+
+
+@python_2_unicode_compatible
+class Photo(models.Model):
+    file = models.ImageField(upload_to='photo_files/%Y-%m-%d')
+    title = models.CharField(max_length=128,
+                             help_text="What is your photo's title?")
+    description = models.TextField(help_text="Describe your photo.")
+    date_uploaded = models.DateField(auto_now_add=True)
+    date_modified = models.DateField(auto_now=True)
+    date_published = models.DateField(auto_now=True)
+
+    CHOICES = (
+        ('Public', 'Public'),
+        ('Private', 'Private'),
+        ('Shared', 'Shared'),
+    )
+    published = models.CharField(max_length=8, choices=CHOICES,
+                                 default='Private')
```

Comment:
Keep all field attributes together.  Move the `CHOICES` constant up to the module level, so you can share the same list between this and the album model.